test/v1alpha1/service.go

```diff
@@ -143,44 +131,50 @@ func CreateRunLatestServiceReady(t pkgTest.TLegacy, clients *test.Clients, names
 
 	t.Log("Waiting for Service to transition to Ready.", "service", names.Service)
 	if err = WaitForServiceState(clients.ServingAlphaClient, names.Service, IsServiceReady, "ServiceIsReady"); err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 
 	t.Log("Checking to ensure Service Status is populated for Ready service")
 	err = validateCreatedServiceStatus(clients, names)
 	if err != nil {
-		return nil, nil, err
-	}
-
-	var httpsTransportOption *spoof.TransportOption
-	if https {
-		tlsOptions := &istiov1alpha3.Server_TLSOptions{
-			Mode:              istiov1alpha3.Server_TLSOptions_SIMPLE,
-			PrivateKey:        "/etc/istio/ingressgateway-certs/tls.key",
-			ServerCertificate: "/etc/istio/ingressgateway-certs/tls.crt",
-		}
-		servers := []*istiov1alpha3.Server{{
-			Hosts: []string{"*"},
-			Port: &istiov1alpha3.Port{
-				Name:     "standard-https",
-				Number:   443,
-				Protocol: "HTTPS",
-			},
-			Tls: tlsOptions,
-		}}
-		httpsTransportOption, err = setupHTTPS(t, clients.KubeClient, names.URL.Host)
-		if err != nil {
-			return nil, nil, err
-		}
-		setupGateway(t, clients, servers)
+		return nil, err
 	}
 
 	t.Log("Getting latest objects Created by Service")
 	resources, err := GetResourceObjects(clients, *names)
 	if err == nil {
 		t.Log("Successfully created Service", names.Service)
 	}
-	return resources, httpsTransportOption, err
+	return resources, err
+}
+
+// GetTransportOption returns transport option
+func GetTransportOption(t pkgTest.TLegacy, clients *test.Clients, https bool) spoof.TransportOption {
```

Comment:
This is very imprecise name. Can we say _what_ kind of transport option is being returned?

Reply:
Yes, agree. Updated to `AddRootCAtoTransport`.